Remotion/Web/Development.WebTesting/DownloadInfrastructure/NullDownloadHelper.cs

```diff
@@ -0,0 +1,31 @@
+﻿// SPDX-FileCopyrightText: (c) RUBICON IT GmbH, www.rubicon.eu
+// SPDX-License-Identifier: LGPL-2.1-or-later
+using System;
+
+namespace Remotion.Web.Development.WebTesting.DownloadInfrastructure;
+
+/// <summary>
+/// Implements the null object pattern for <see cref="IDownloadHelper"/>.
+/// </summary>
+public class NullDownloadHelper : IDownloadHelper
+{
+  public static readonly NullDownloadHelper Instance = new();
+
+  private NullDownloadHelper ()
+  {
+  }
+
+  public IDownloadedFile HandleDownloadWithExpectedFileName (string fileName, TimeSpan? downloadStartedTimeout = null, TimeSpan? downloadUpdatedTimeout = null)
+  {
+    throw new NotSupportedException();
+  }
+
+  public IDownloadedFile HandleDownloadWithDetectedFileName (TimeSpan? downloadStartedTimeout = null, TimeSpan? downloadUpdatedTimeout = null)
+  {
+    throw new NotSupportedException();
+  }
```

Comment:
drop the default parameter values? don't help on concrete types anyway when it's always used via interface